UIMenu.cs

```diff
@@ -476,6 +482,37 @@ public void Clear()
             RecaulculateDescriptionPosition();
         }
 
+        private void DrawCalculations()
+        {
+            drawWidth = new Size(431 + WidthOffset, 107);
+
+            safe = GetSafezoneBounds();
+
+            backgroundSize = Size > MaxItemsOnScreen + 1 ? new Size(431 + WidthOffset, 38 * (MaxItemsOnScreen + 1)) : new Size(431 + WidthOffset, 38 * Size);
+
+            if (!String.IsNullOrWhiteSpace(MenuItems[_activeItem % (MenuItems.Count)].Description))
```

Comment:
That's a DivideByZeroException right here if MenuItems.Count is 0. You should move all item-size independent code to the top of the function and then return in the middle if there are no items.